core/create/src/prompts/main.ts

```diff
@@ -2,35 +2,52 @@ import { styles } from '@dotcom-tool-kit/logger'
 import type { PackageJson } from 'type-fest'
 import { existsSync } from 'fs'
 import prompt from 'prompts'
+import { BizOpsSystem } from '../bizOps'
 
 type PromptNames = 'preset' | 'additional' | 'addEslintConfig' | 'deleteConfig' | 'uninstall'
 
 export interface MainParams {
+  bizOpsSystem?: BizOpsSystem
   packageJson: PackageJson
   originalCircleConfig?: string
   eslintConfigPath: string
 }
 
 export default async ({
+  bizOpsSystem,
   packageJson,
   originalCircleConfig,
   eslintConfigPath
 }: MainParams): Promise<prompt.Answers<PromptNames>> => {
   const isPackageInstalled = (packageName: string) =>
     Object.keys(packageJson.devDependencies ?? {}).includes(packageName)
 
+  const presetChoices = [
+    { title: 'A user-facing (frontend) app', value: 'frontend-app' },
+    { title: 'A Heroku backend app', value: 'backend-heroku-app' },
+    { title: 'A Serverless backend app', value: 'backend-serverless-app' },
+    { title: 'An npm component', value: 'component' }
```

Comment:
**thought:** We should at some point change these to use Biz Ops terminology. `app` -> `system` and `component` -> `package`.

Reply:
it's quite tricky seeing as the plugins use the same terminology 😬